x-pack/plugins/synthetics/public/apps/synthetics/components/monitor_summary/tabs_content/duration_panel.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ReportTypes } from '@kbn/observability-plugin/public';
+import { useParams } from 'react-router-dom';
+import { ClientPluginsStart } from '../../../../../plugin';
+
+export const DurationPanel = () => {
```

Comment:
nit: Shouldn't be named as `*Panel` if it's not going to be used elsewhere on its own